autoware_launch/launch/logging_simulator.launch.xml

```diff
@@ -42,11 +42,14 @@
 
   <!-- Vehicle -->
   <group>
+    <let name="launch_vehicle_interface" value="false" if="$(var vehicle_simulation)"/>
+    <let name="launch_vehicle_interface" value="true" unless="$(var vehicle_simulation)"/>
```

Comment:
IMO, you can make launch_vehicle_interface directly instead of vehicle_simulation